app/styles/_library/_objects.forms.scss

```diff
@@ -55,12 +55,22 @@ div.o-confirm-message p {
 .c-search__button {
   height: 100%;
   padding: 10px;
+  margin: 0px;
+  border-radius: 0px;
+}
+
+.c-search__button:focus {
```

Comment:
We can't just remove the focus outline without providing an alternative focus style.  I've sent the ticket back to design for feedback.

Reply:
And it looks like this was already merged as #75 ? Not sure what to do with this now...

Reply:
Maybe it was, I’m not sure. I saw Brad was offline and that the ticket got approved and matched it to this open branch.

Reply:
guess i can just release with the bug for now to get things out and leave the pr and jira ticket open.

Reply:
late to the game here -- i'll follow up with design/product to resolve this. the focus border was being flagged as bug so i took it out.